packages/terra-clinical-action-header/tests/wdio/action-header-spec.js

```diff
@@ -0,0 +1,95 @@
+/* global browser, Terra, before */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+
+describe('Action Header', () => {
+  describe('Displays a default action header', () => {
+    before(() => {
+      browser.url('/#/tests/action-header-tests/action-header');
```

Comment:
change to `browser.url('/#/raw/tests/clinical-action-header/default-action-header');`